File: src/models/DeveloperCount.js
import { Schema, model} from "mongoose";

const DeveloperCountsSchema = new Schema({
    total: {
        type: Number,
        required: true,
        default: 0,
    },
    mean: {
        type: Number,
        required: true,
        default: 0,
    },
    mode: {
        type: Number,
        required: true,
        default: 0,
    },
    median: {
        type: Number,
        required: true,
        default: 0,
    },
    date: {
        type: Date,
        required: true,
        default: Date.now,
    },
    isDaily: {
        type: Boolean,
        default: false,
    },
});

export const DeveloperCountsModel = model("DeveloperCounts", DeveloperCountsSchema);

File: SnapstatsOrg/Shared/Models/DeveloperCount.cs
﻿using System;

namespace SnapstatsOrg.Shared.Models
{
    public class DeveloperCount
    {
        public int total { get; set; }
        public double mean { get; set; }
        public int median { get; set; }
        public int mode { get; set; }
        public DateTime date { get; set; }
    }
}
